ckan_cloud_operator/labels/manager.py

```diff
@@ -11,6 +11,7 @@ def initialize(log_kwargs=None):
 
 def get_label_prefix(short=False):
     """Returns a global label prefix which should be used to namespace operator objects"""
+    return 'ckan-cloud'
```

Comment:
I assume this is hardcoded for development purposes... (Or we don't need another `return` coming after)